src/quo/components/drawers/drawer_action/view.cljs

```diff
@@ -0,0 +1,86 @@
+    [quo.components.selectors.selectors.view :as selectors]
+    [quo.theme]
+    [react-native.core :as rn]
+    [schema.core :as schema]))
+
+(def ?schema
+  [:=>
+   [:cat
+    [:map
+     [:action {:optional true} [:maybe [:enum :arrow :toggle]]]
+     [:icon {:optional true} [:maybe :keyword]]
+     [:description {:optional true} [:maybe :string]]
+     [:state {:optional true} [:maybe [:enum :selected]]]
+     [:title {:optional true} :string]
+     [:on-press {:optional true} [:maybe fn?]]
+     [:customization-color {:optional true}
+      [:maybe :schema.common/customization-color]]
+     [:theme {:optional true} [:maybe :schema.common/theme]]
+     [:blur? {:optional true} [:maybe :boolean]]]]
+   :any])
+
+(defn view-internal
+  [{:keys [action icon description state title on-press
+           customization-color theme blur?]
+    :or   {customization-color :blue
+           blur?               false}}]
+  (let [[pressed? set-pressed] (rn/use-state false)
+        on-press-in            #(set-pressed true)
+        on-press-out           #(set-pressed false)]
+    [rn/pressable
+     {:on-press            on-press
+      :on-press-in         on-press-in
+      :on-press-out        on-press-out
+      :style               (style/container {:state               state
+                                             :action              action
+                                             :customization-color customization-color
+                                             :theme               theme
+                                             :pressed?            pressed?
+                                             :description?        (not-empty description)
+                                             :blur?               blur?})
+      :accessibility-label :container}
+     (when icon
+       [icon/icon icon
+        (style/left-icon {:theme theme
+                          :blur? blur?})])
+     [rn/view (style/text-container)
+      [text/text {:weight :medium}
+       title]
+
+      (when (seq description)
+        [text/text
+         {:size :paragraph-2
+          :style
+          (style/desc {:theme theme
+                       :blur? blur?})}
+         description])]
+
+     (cond
+       (= action :toggle)
+       [selectors/view
+        {:theme               theme
+         :label-prefix        "toggle"
+         :customization-color customization-color
+         :type                :toggle
+         :checked?            (= state :selected)
+         :blur?               blur?}]
+
+       (= action :arrow)
+       [icon/icon :i/chevron-right
+        (style/arrow-icon {:theme theme
+                           :blur? blur?})]
+
+       (= state :selected)
+       [icon/icon :i/check
+        (style/check {:theme               theme
+                      :blur?               blur?
+                      :customization-color customization-color})])]))
+
+(def view
+  (quo.theme/with-theme
```

Comment:
please remove `(quo.theme/with-theme` wrapper and use `quo.theme/use-theme-value` directly i the view